godot-macros/src/class/godot_api.rs

```diff
@@ -67,21 +67,35 @@ impl BoundAttr {
     }
 }
 
+/// Holds information known from a signal's definition
+struct SignalDefinition {
+    /// The signal's function signature.
+    signature: Function,
+
+    /// The signal's non-gdext attributes (that is, excluding #[signal]).
```

Comment:
This could be understood as "non-gdext attributes except `#[signal]`", maybe make it very clear with "The signal's non-gdext attributes (everything except #[signal])".

Maybe it's just me being a native speaker, but better to have it unambiguous 😉 

Same above with `#[func]`.